app/controllers/miq_policy_controller/policies.rb

```diff
@@ -226,7 +226,8 @@ def excluded_event?(event)
 
   def policy_get_all_folders(parent = nil)
     if !parent.nil?
-      @folders = MiqPolicy::UI_FOLDERS.collect(&:name)
+      # TODO: this is dead code, the one caller passes no arguments
```

Comment:
I agree. Could you remove the todo and submit a separate pr removing the dead code?
It isn't urgent but would be nice (please do not depend on that pr if you do)